content/docs/04.5-devx.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Palette Dev Engine"
+metaTitle: "Palette Dev Engine "
+metaDescription: "Explore Palette Dev Engine"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Palette provides a developer persona of the Palette operations in addition to the DevOps persona of creating, deploying, and managing applications in Palette. This persona is called Palette Dev Engine (PDE); the experience developers have while building, maintaining, testing, deploying, and monitoring applications through Palette. A developer can easily switch between these two personas - `App Mode and Cluster Mode`- through the Palette UI. We offer the PDE from two wide perspectives:
+
+<br />
+
+
+# Quick Start with Palette Dev Engine
+
+The quick-start developer experience with Palette Dev Engine is exceptionally straightforward. We provide a smooth developer experience where anyone can log in to Palette Dev Engine and deploy their first application in just **three** steps.
+
+|    |  |
+|----|---|
+|**Step 1** |[Login to Palette Dev Engine](/devx#loginoptions)|
+|**Step 2** |Select an App Profile (we do offer a set of out-of-the-box profiles) or [Create an App Profile](/devx/app-profile) based on your use case|
+|**Step 3** |Using the App Profile [deploy your application](/devx/apps)| 
+
+Refer the below links to explore the Palette Dev Engine in depth ....
+
+<br />
+
+* **Sign up to Manage Apps**: As a free tier offering from Palette for Developers to [Land and Expand](/devx/dev-land-explore).
+
+
+* **Sign up to Manage Clusters**: For [Enterprise Developers](/devx/enterprise-user) full Palette capabilities.
+
+# Login Options
+
+Palette provides two options of developer log in to the Palette Dev Engine console:
```

Comment:
Palette provides two options for developers to log in...

Reply:
done